README.md

```diff
@@ -319,6 +319,77 @@ awslocal kinesis list-streams
 **UPDATE**: Use the environment variable `$LOCALSTACK_HOSTNAME` to determine the target host
 inside your Lambda function. See [Configurations](#Configurations) section for more details.
 
+## Using the official AWS CLI version 2 Docker image with Localstack Docker container
+
+By default the the container running [amazon/aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-docker.html) is isolated from `0.0.0.0:4566` on the host machine, that means that aws-cli cannot reach localstack through your shell.
+
+To ensure that the two docker containers can communicate create a network on the docker engine:
+
+```bash
+$ ▶ docker network create localstack
+0c9cb3d37b0ea1bfeb6b77ade0ce5525e33c7929d69f49c3e5ed0af457bdf123
+```
+Then modify the `docker-compose.yml` specifying the network to use:
+
+```yml
+version: "3.8"
+
+networks:
+  default:
+    external:
+      name: "localstack"
+
+services:
```

Comment:
Thanks for extending the docs @lucasepe . I think a few sections are a bit redundant - can we try to trim it down to the bare minimum (and avoid duplication with other parts of the README)? I think (at least) lines 335-336, 341-360, 363-368 are redundant and can probably be removed. Thanks!